### ISSUE TITLE
在自己的数据集上进行了测试，发现对于不同篡改类型的效果 差距很大

### ISSUE THREAD

**Issue by ff202118** [SunnyHaze/IML-ViT-IMDLBenCo#2]
又来打扰你啦。具体来说，我测试了Remove和Copy-Move两种篡改类型。Remove训练数量为900左右，Copy-Move训练数量为3000左右，两者的图像尺寸都在250px*250px左右。两者的参数均如下所示，但是Remove的像素级F1分数只有0.0296，而Copy-Move可以达到0.871，这种差距是因为模型本身吗？另外，我该怎么查看图像级的指标（是不是恢复图片中的注释即可？）
base_dir="./output_dir"
mkdir -p ${base_dir}

CUDA_VISIBLE_DEVICES=0 \
torchrun  \
    --standalone    \
    --nnodes=1     \
    --nproc_per_node=1 \
./train.py \
    --model IMLViTModel \
    --vit_pretrain_path ./pretrained-weights/mae_pretrain_vit_base.pth \
    --world_size 1 \
    --batch_size 2 \
    --data_path train1.json \
    --epochs 200 \
    --lr 1e-4 \
    --image_size 1024 \
    --if_padding \
    --min_lr 5e-7 \
    --weight_decay 0.05 \
    --edge_mask_width 7 \
    --test_data_path "test_datasets.json" \
    --warmup_epochs 2 \
    --output_dir ${base_dir}/ \
    --log_dir ${base_dir}/ \
    --accum_iter 8 \
    --seed 42 \
    --test_period 4 \
    --edge_lambda 40 \
2> ${base_dir}/error.log 1>${base_dir}/logs.log

![Image](https://github.com/user-attachments/assets/f5db0758-7c71-418a-a6da-549f2d1ec018)
